### ISSUE TITLE
package `marked` doesn't satisfy `marked-terminal` peer-dependency

### ISSUE THREAD

**Issue by hikaruna** [firebase/firebase-tools#1560]
### [REQUIRED] Environment info

**firebase-tools:  "7.2.1",

**Platform:linux(debian stretch on {docker FROM node:8.16.0-stretch on mac} )

### [REQUIRED] Test case

see Steps to reproduce

### [REQUIRED] Steps to reproduce

1. step1
    1. boot mac
    1. install docker desctop for mac
    1. launch docker (in mac)
    1. docker run -it --rm node:8.16.0-stretch bash
    1. mkdir workspace ; cd workspace
1. step2
    1. npm init -y
    1. npm i -D firebase-tools
    

### [REQUIRED] Expected behavior

```npm i -D firebase-tools```  stdout does not contain 

```npm WARN marked-terminal@3.2.0 requires a peer of marked@^0.4.0 || ^0.5.0 || ^0.6.0 but none is installed. You must install peer dependencies yourself.```

---

firebase-tools requires  "marked": "^0.7.0", "marked-terminal": "^3.1.1"
and
marked-terminal:3.1.1 require "marked": "^0.4.0 || ^0.5.0 || ^0.6.0".

I think firebase-tools should use marked: 0.6.0 until mark-terminal use mark: 0.7.0```
firebase-tools require  "marked": "^0.7.0", "marked-terminal": "^3.1.1",

### [REQUIRED] Actual behavior

```npm i -D firebase-tools``` stdout is

```
...
npm WARN marked-terminal@3.2.0 requires a peer of marked@^0.4.0 || ^0.5.0 || ^0.6.0 but none is installed. You must install peer dependencies yourself.
...
```

**Comment by bkendall** [comment]
Yes, this warning is printed, and it's known at this time. There shouldn't be any issues using `firebase-tools`, but it will print this warning. `marked` had a [security advisory published](https://www.npmjs.com/advisories/1076) and published the `0.7` updated. `marked-terminal` (which we also use) has to explicitly whitelist every `0.x` minor version of `marked` in order to support it. I have a [PR out to update `marked-terminal`](https://github.com/mikaelbr/marked-terminal/pull/54) (but I have no power there).

**Comment by bkendall** [comment]
(I'll leave this but open until this issue is fixed, at any rate)
